<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverviewFrame</class>
 <widget class="QFrame" name="OverviewFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>

  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">

   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">

       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>    


       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>   
         <item row="0" column="0">
          <widget class="QFrame" name="m_overviewWalletFrame">
           <property name="minimumSize">
            <size>
             <width>846</width>
             <height>592</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">margin: 0;</string>
           </property>

             <layout class="QVBoxLayout" name="verticalLayout2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>


              <item>

               <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,0">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>846</width>
                  <height>200</height>
                 </rect>
                </property>

                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <property name="horizontalSpacing">
                  <number>0</number>
                 </property>
                 <property name="verticalSpacing">
                  <number>10</number>
                 </property>

                  <item row="1" column="0">
                   <widget class="QLabel" name="m_actualBalanceTextLabel">
                    <property name="text">
                     <string>Available Balance:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLabel" name="m_actualBalanceLabel">
                    <property name="font">
                     <font>
                      <weight>75</weight>
                      <bold>true</bold>

                     </font>
                    </property>
                    <property name="text">
                     <string notr="true">0.00</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="styleSheet">
                     <string>padding-right:5;</string>
                    </property>                
                   </widget>
                  </item>
                  <item row="1" column="2">
                   <widget class="QLabel" name="m_tickerLabel1">
                    <property name="text">
                     <string notr="true"/>
                    </property>
                   </widget>
                  </item>

                  <item row="2" column="0">
                   <widget class="QLabel" name="m_pendingBalanceTextLabel">
                    <property name="text">
                     <string>Pending Transactions:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLabel" name="m_pendingBalanceLabel">
                    <property name="text">
                     <string notr="true">0.00</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="styleSheet">
                     <string>padding-right:5;</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="2">
                   <widget class="QLabel" name="m_tickerLabel2">
                    <property name="text">
                     <string notr="true"/>
                    </property>
                   </widget>
                  </item>

                  <item row="3" column="0">
                   <widget class="QLabel" name="m_messagesCountTextLabel">
                    <property name="text">
                     <string>Messages:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1" colspan="2">
                   <widget class="QLabel" name="m_messagesCountLabel">
                    <property name="font">
                     <font>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string notr="true">0</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>

                  <item row="4" column="0">
                   <widget class="QLabel" name="m_walletEncryptedTextLabel">
                    <property name="text">
                     <string>Wallet Password Encryption:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="1" colspan="2">
                   <widget class="QLabel" name="m_walletEncryptedLabel">
                    <property name="font">
                     <font>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string notr="true">Off</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="styleSheet">
                     <string>color:red;</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="2">
                   <widget class="QLabel" name="m_walletEncryptedLabel2">
                    <property name="text">
                     <string notr="true"/>
                    </property>
                   </widget>
                  </item>

                  <item row="5" column="0">
                   <widget class="QLabel" name="m_peersCountTextLabel">
                    <property name="text">
                     <string>Connected Peers:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="5" column="1" colspan="2">
                   <widget class="QLabel" name="m_peersCountLabel">
                    <property name="font">
                     <font>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string notr="true">0</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="styleSheet">
                     <string>color:black;</string>
                    </property>
                   </widget>
                  </item>
                  <item row="5" column="2">
                   <widget class="QLabel" name="m_peersCountLabel2">
                    <property name="text">
                     <string notr="true"/>
                    </property>
                   </widget>
                  </item>

               </layout>

              </item>

             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>


              <item>
                <widget class="QLabel" name="m_getStartedLabel">
                  <property name="text">
                    <string>Tips on getting started:

1 - Create a password: Secure your wallet by enabling password encryption. Go to settings to encrypt your wallet.
2 - Backup your wallet: You need your wallet data to access your sent messages. Make sure to frequently backup it up.
3 - Get coins: Mining helps maintain the blockchain and rewards you with coins! Start mining now. 
4 - Give us feedback: We love to hear about your experience. Tell us what you think.
                    </string>
                   </property>
                </widget>                                   
              </item>




            </layout>


          </widget>
         </item>
        </layout>
       </item>

    </layout>
  </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>encryptWallet()</slot>
 </slots>

</ui>
